test/server/index.test.ts

```diff
@@ -1,12 +1,81 @@
-import { C } from "./../../src/server";
```

Comment:
**Annotation:** Remove temporary code and test for server index. This includes App and Port.